### PR TITLE
Ft send invite 150397539

### PR DESCRIPTION
**Pull request by Dannytebj**
#### What does this PR do?
This PR adds the search users and send invite feature.
#### Description of Task to be completed?
Have the following endpoints working:
```
GET     /api/users/search
POST /api/users/sendInvites
```
#### How should this be manually tested?
PostMan
#### What are the relevant pivotal tracker stories?
- Users should be able to send invites to at most 11 people other than themselves
#### Screenshots (if appropriate)
screen shot 2017-09-08 at 10 16 55 pm
screen shot 2017-09-08 at 10 17 45 pm

### CODE REVIEW

File: app/views/includes/foot.jade [hunk 1]
```diff
@@ -5,7 +5,7 @@ script.
       s.parentNode.insertBefore(g,s)}(document,'script'));
 script(type='text/javascript', src='/lib/jquery/jquery.js')
 script(type='text/javascript', src='/lib/underscore/underscore-min.js')
-
+// Bootstrap
```
Comment (mradeybee) [inline app/views/includes/foot.jade]: Remove this if its not doing anything there.

File: app/views/includes/head.jade [hunk 2]
```diff
@@ -34,8 +34,7 @@ head
   link(rel='stylesheet', href='/css/style.css')
   link(rel="stylesheet", href="/css/main.css")
   link(rel = "stylesheet", href = 'https://fonts.googleapis.com/icon?family=Material+Icons')
-
-
+  // Bootstrap
```
Comment (mradeybee) [inline app/views/includes/head.jade]: Ditto Remove

File: config/middlewares/authentication.js [hunk 3]
```diff
@@ -23,4 +23,4 @@ module.exports = (req, res, next) => {
       })
       .catch(err => res.status(404).json(err));
   });
-};
\ No newline at end of file
+};
```
Comment (mradeybee) [inline config/middlewares/authentication.js:26]: Always add a new line at the end of files

**Review by mradeybee — CHANGES_REQUESTED** [review]
Just a few comments, this is almost ready.

File: config/middlewares/authentication.js (hunk 3 above)
Comment (Dannytebj) [reply config/middlewares/authentication.js:26]: Okay Thanks, I will implement the feedback and raise another PR.

**Pull request closed without merging**